src/flag_gems/ops/max.py

```diff
@@ -93,6 +93,8 @@ def max_kernel(
 
 def max(inp):
     logging.debug("GEMS MAX")
+    if not inp.is_contiguous():
+        inp = inp.contiguous()
```

Comment:
Thanks very much.
There is a contiguous check in `contiguous()`, so just `inp = inp.contiguous()` is enough.

Reply:
done